### PR TITLE
Added MUCommander

### PR DESCRIPTION
**Pull request by Xmetalfanx**
Closes https://github.com/wimpysworld/deb-get/issues/336

tested and installs and runs locally

### CODE REVIEW

File: deb-get [hunk 1]
```diff
@@ -2413,6 +2413,17 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_mucommander() {
+    get_github_releases "https://api.github.com/repos/mucommander/mucommander/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL="$(grep "browser_download_url.*${ARCHS_SUPPORTED}.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
```
[the remark this answers is not in the document]
Comment (Xmetalfanx) [reply deb-get]: yes i did and updated (sure i made this mistake in another branch too) ... the logic makes so much more sense this way

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
Please drop the Virtualbox change in README.md, rebase and squash the commits.

**Comment by Xmetalfanx** [comment]
https://github.com/Xmetalfanx/deb-get/commit/5b8c8773ebadbb3af0996a2495aea01a6f39a929 not sure if github is showing the correct link above this comment after i changed what was asked and force-pushed after squashing the fix, but this link here is what i have now ... 

is this good, or is it better to close this PR (I will leave it open unles suggested otherwise) and make a new PR or just better to leave things "as-is"?

**Comment by natanjunges** [comment]
The format of package definitions has changed. Please read [01-main/CONTRIBUTING](https://github.com/wimpysworld/deb-get/blob/main/01-main/CONTRIBUTING.md) and [EXTREPO](https://github.com/wimpysworld/deb-get/blob/main/EXTREPO.md), update your implementation and submit a new PR.

**Pull request closed without merging**